src/pages/workspace/categories/CategorySettingsPage.tsx

```diff
@@ -49,6 +54,20 @@ function CategorySettingsPage({route, policyCategories}: CategorySettingsPagePro
         Navigation.navigate(ROUTES.WORKSPACE_CATEGORY_EDIT.getRoute(route.params.policyID, policyCategory.name));
     };
 
+    const removeCategory = () => {
```

Comment:
use `useCallback`

Reply:
Why callback here? The confirm modal isn't memoized, and the onCancel is inline function anyway. The callback seems extra. If we wrap the confirm modal with memo, that might work better, especially if we use memoization and pass callbacks in onConfirm and onCancel.

Reply:
ok sounds good to me